### PR TITLE
Added social links and footer

### PR DESCRIPTION
**Pull request by saarthakchats**
## Added social links and footer

> @0x48piraj worked on the `footer.html`

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
 	</div>
+	<footer style="color: black">
+		<font color="white">
+			<a href="facebook.com/fossasia">Facebook</a>
+			<a href="twitter.com/fossasia">Twitter</a>
+			<a href="github.com/fossasia"></a>
+		</font>
+		
+
+
```
Comment (harshit98) [inline _includes/footer.html]: Don't leave too much space here.

**Comment by harshit98** [comment]
@saarthakchats Provide screenshot too. Thanks!

**Comment by saarthakchats** [comment]
ll
All checks have passed and I have removed that empty space. Please merge and also approve my GCI task

**Comment by saarthakchats** [comment]
Fine, I made a new PR.
https://github.com/fossasia/gci17.fossasia.org/pull/195
I have only changed the picture in that. Please merge that.

**Comment by harshit98** [comment]
@saarthakchats I'm not able to find the social link in your screenshot.

**Comment by saarthakchats** [comment]
Sorry, 
foot
I attached the wrong screenshot. This is it

**Comment by saarthakchats** [comment]
Now can you approve my task?

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
@harshit98 Please see and merge if it looks appropriate. :+1:

**Comment by akshatnitd** [comment]
@saarthakchats I have merged the PR. Make sure to squash the commits from next time :)

**Comment by mariobehling** [comment]
I cannot see it on the site deployed.

**Comment by mariobehling** [comment]
There is no relevant issue available for this PR.

**Comment by akshatnitd** [comment]
@mariobehling  I realised now that, this wasn't supposed to be merged. 

@saarthakchats There is no relevant issue for this PR. Also, you have attached misleading screenshot above when asked for. I am reverting this PR. Also, make sure to give a demo link of the PR, which helps the reviewers and prevents from incorrect merges. Also, the links you have added are incorrect.

**Pull request merged**